frontend/public/locales/en/translation.json

```diff
@@ -371,6 +371,8 @@
   "access.users": "Users",
   "access.usersInGroup": "Users in group",
   "access.usersInGroup.view": "View users in group",
+  "acm.plugin.not.ready": "The <bold>Red Hat Advanced Cluster Management for Kubernetes</bold> operator is installed, but the console plugin is not available. If operator installation has just completed, an alert should appear in a few moments prompting you to refresh the web console to access the latest version.",
```

Comment:
Suggestion
```suggestion
  "acm.plugin.not.ready": "The <bold>Red Hat Advanced Cluster Management for Kubernetes</bold> operator is installed, but the console plugin is not available. If the operator installation completes, an alert appears in a few moments prompting you to refresh the web console to access the latest version.",
```

Reply:
@KevinFCormier thanks for explaining your perspective and providing a suggestion. Your suggestion looks good to me!